src/decoder_fmmidi.cpp

```diff
@@ -123,6 +125,7 @@ int FmMidiDecoder::FillBuffer(uint8_t* buffer, int length) {
 		seq->play(mtime, this);
 		notes = synthesize(reinterpret_cast<int_least16_t*>(buffer), samples, frequency);
 		mtime += delta;
+		ticks += ticks_per_sec * delta;
```

Comment:
Had to change this to float because ticks_per_sec and delta are float so the error increases over time when using int.